core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -237,8 +238,8 @@ contract Liquidatable is PricelessPositionManager {
     function dispute(uint id, address sponsor) external disputable(id, sponsor) fees() {
         LiquidationData storage disputedLiquidation = _getLiquidationData(sponsor, id);
 
-        FixedPoint.Unsigned memory disputeBondAmount = _getCollateral(disputedLiquidation.rawLockedCollateral).mul(
-            disputeBondPct
+        FixedPoint.Unsigned memory disputeBondAmount = disputedLiquidation.lockedCollateral.mul(disputeBondPct).mul(
```

Comment:
Just to understand better, you swapped `_getCollateral(disputedLiquidation.lockedCollateral)` for `disputedLiquidation.lockedCollateral` because you want to charge the dispute bond **as a % of the pre-liquidation-fee collateral? (i.e. the amount of collateral snapshotted at beginning of liquidation)**?

Reply:
Nope, the return value from this math should be the same as it was previously. Because the rawLockedCollateral took into account the fees, but now the rawFeeAttenuation takes into account the fees. In both cases, it's computing a % of the collateral that is left after charging fees *up to that point*. Does that make sense?